apps/docs/app/routes/_docs/profil/farger.tsx

```diff
@@ -0,0 +1,57 @@
+import { createFileRoute } from '@tanstack/react-router';
+
+export const Route = createFileRoute('/_docs/profil/farger')({
+  component: RouteComponent,
+});
+
+function getAllColors() {
+  const cssVariables: Record<string, string> = {};
+
+  for (const sheet of document.styleSheets) {
+    for (const rule of sheet.cssRules ?? []) {
+      if (rule instanceof CSSStyleRule && rule.selectorText === ':root') {
+        for (const style of rule.style) {
+          const prefix = '--gm-color-';
+          if (style.startsWith('--gm-color-')) {
+            cssVariables[style.replace(prefix, '')] = rule.style
+              .getPropertyValue(style)
+              .trim();
+          }
+        }
+      }
+    }
+  }
+
+  return cssVariables;
+}
+
+function RouteComponent() {
+  const colors = getAllColors();
```

Comment:
Dette er en sideeffect som gjør at render ikke lenger er pure. Det vil heller ikke funke med SSR.

Men i stedet for å fikse det kan vi heller se på å få til å lese det ut fra tailwind configen.

Reply:
Stemmer, det tenkte jeg ikke på! 😓 

Men ja, gjerne det! Har prøvd iherdig å få det til, men klarer det ikke 😞